src/content/docs/validator/external-chains/xrpl.mdx

```diff
@@ -199,3 +196,6 @@ For your node to be recognized by the Axelar network, you need to register your
 ampd register-chain-support amplifier xrpl
 ```
 
+## Troubleshooting
+
+If your node runs into an issues (such as an `AppHash` error) the node orchestration should perform a `rollback` on crash **before restarting the node** to auto-recover from such errors. The rollback can be performed by running `axelard rollback`. 
```

Comment:
Make it clear that it's the Axelar node that ampd points to that's being referred here, and not the XRPL node that most of this dlc focuses on